frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -107,6 +113,41 @@ local highlight_style = {
     invert = _("Invert"),
 }
 
+local function getPanelZoomSupportedExt()
+    local default_supported_ext = {
+        cbz = true,
+        cbt = true,
+    }
+    return G_reader_settings:readSetting("panel_zoom_ext") or default_supported_ext
+end
+
+local function isDocumentComicOrManga(file)
```

Comment:
Unless you plan later to detect if document is really a comic or manga, I would rename this to what it actually does :) 
`isPanelZoomSupported(file)`